From: xie xingguo <xie.xingguo@zte.com.cn>
Date: Sun, 18 Sep 2016 10:33:56 +0800
Subject: tools/rebuild_mondb: avoid unnecessary result code cast

In general we return negative codes for error cases, so there is
no need perform the cast here.

Signed-off-by: xie xingguo <xie.xingguo@zte.com.cn>
(cherry picked from commit 6a1c01d334fe65124043aa68a6e0cfaea43836b5)

Resolves: rhbz#1374224
---
 src/tools/rebuild_mondb.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tools/rebuild_mondb.cc b/src/tools/rebuild_mondb.cc
index a52577d..a8d36ec 100644
--- a/src/tools/rebuild_mondb.cc
+++ b/src/tools/rebuild_mondb.cc
@@ -22,7 +22,7 @@ int update_mon_db(ObjectStore& fs, OSDSuperblock& sb,
   int r = ms.create_and_open(cerr);
   if (r < 0) {
     cerr << "unable to open mon store: " << store_path << std::endl;
-    return EINVAL;
+    return r;
   }
   if ((r = update_auth(keyring, sb, ms)) < 0) {
     goto out;
